modules/data-table/main/index.coffee

```diff
@@ -837,3 +1204,4 @@ hx.dataTable = (options) ->
 
 hx.dataTable.objectFeed = objectFeed
 hx.dataTable.urlFeed = urlFeed
+hx.dataTable.getAdvancedSearchFilter = getAdvancedSearchFilter
```

Comment:
I've exposed the function for getting the advanced search filter as it would be a useful thing to have when writing a custom feed. I'm not sure if it's required though.